demand-dashboard/app/views/dashboardAnalysis.scala.html

```diff
@@ -0,0 +1,111 @@
+@(mapboxAccessToken: String)
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Analysis Dashboard</title>
+    <link rel="stylesheet" href="@routes.Assets.versioned("main.css")">
+    <link href="https://api.mapbox.com/mapbox-gl-js/v0.28.0/mapbox-gl.css" rel="stylesheet" type="text/css"/>
+    <link href="https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-directions/v3.0.2/mapbox-gl-directions.css" rel="stylesheet" type="text/css"/>
+    <script src="@routes.Assets.versioned("lib/jquery/jquery.min.js")"></script>
+        <script src="https://api.mapbox.com/mapbox-gl-js/v0.28.0/mapbox-gl.js"></script>
+    <script src="https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-directions/v3.0.2/mapbox-gl-directions.js"></script>
+    <script src="https://api.tiles.mapbox.com/mapbox.js/plugins/turf/v2.0.0/turf.min.js" charset="utf-8"></script>
+    <script src="@routes.Assets.versioned("analysis.js")"></script>
+
+    <!-- jQuery UI Slider -->
+    <link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
```

Comment:
Let's use a WebJar for this.  Find the dependency on http://webjars.org add it to `build.sbt` and since there is a transitive dep on jquery you can remove it from `build.sbt`